lang_qc/db/helper/qc.py

```diff
@@ -132,6 +133,38 @@ def qc_state_for_product_exists(
     return bool(session.execute(query).scalar_one())
 
 
+def qc_state_for_products_exists(
```

Comment:
I don't like that `qc_state_for_products_exists` and `qc_state_for_product_exists` are so similar in name and the difference is in the fourth word.

Perhaps we could enhance readability by renaming to `product_has_qc_state` and `products_have_qc_state`? Or some other idea that differentiates them better.